data/tribes/initialization/empire/units.lua

```diff
@@ -1147,24 +1155,43 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_warehouse",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Warehouse
+            lore = pgettext("empire_building", "‘Bis dat, qui cito dat, nil dat, qui munera tardat.’ (Who gives promptly, gives twice, who gives late, gives nothing.)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Warehouse
+            lore_author = pgettext("proverb written over the doors of a warehouse"),
             -- TRANSLATORS: Purpose helptext for an empire warehouse: Warehouse
             purpose = pgettext("empire_building", "Your workers and soldiers will find shelter here. Also stores your wares and tools.")
          }
       },
       {
          name = "empire_port",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Port
+            lore = pgettext("empire_building", "‘Navigare necesse est.’ (To sail is necessary.)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Port
+            lore_author = pgettext("Admiral Litus Marius Exsiccatus"),
             -- TRANSLATORS: Purpose helptext for an empire warehouse: Port
             purpose = pgettext("empire_building", "Serves as a base for overseas colonization and trade. Also stores your soldiers, wares and tools."),
             -- TRANSLATORS: Note helptext for an empire warehouse: Port
             note = pgettext("empire_building", "Similar to the Headquarters a Port can be attacked and destroyed by an enemy. It is recommendable to send soldiers to defend it.")
          }
       },
 
+--[[ lore template
+            -- TRANSLATORS: Lore helptext for an empire production site:
+            lore = pgettext("empire_building", "‘’ ()"),
+            -- TRANSLATORS: Lore author helptext for an empire production site:
+            lore_author = pgettext("???"),
+]]--
+
       -- Small
       {
          name = "empire_quarry",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Quarry
+            lore = pgettext("empire_building", "‘Lapides at ædificium.’ (Stones for Buildings.)"),
```

Comment:
@klaus-halfmann Are you sure of this? 'At' seems to have different meaning.

Reply:
```suggestion
            lore = pgettext("empire_building", "‘Lapides pro ædificia.’ (Stones for buildings.)"),
```
Not 100% sure, I'm not a native speaker :wink:

Also not sure if we should use the `æ` character or `ae`? The latter is what I'm used to seeing in Latin texts.

Reply:
I think the Romans *did* use the Æ ligarure. And I think it looks cool and special, that's why I overuse it on purpose. :smile:

I think this building needs a better quotation anyway.